2025-07-28 08:48:17,578 ERROR: Failed to load logs: (1054, "Unknown column 'user_email' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:310]
2025-07-28 09:56:56,851 ERROR: Failed to load logs: (1054, "Unknown column 'log_level' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:310]
2025-07-28 09:59:56,880 ERROR: Failed to load logs: (1054, "Unknown column 'log_level' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:310]
2025-07-28 10:00:02,540 ERROR: Failed to load logs: (1054, "Unknown column 'log_level' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:310]
2025-07-28 10:12:51,361 ERROR: Failed to load logs: (1054, "Unknown column 'log_level' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
2025-07-28 10:16:46,118 ERROR: Failed to load logs: (1054, "Unknown column 'log_message' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
2025-07-28 10:16:47,504 ERROR: Failed to load logs: (1054, "Unknown column 'log_message' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
2025-07-28 10:17:45,597 ERROR: Failed to load logs: (1054, "Unknown column 'endpoint' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
2025-07-28 10:17:46,703 ERROR: Failed to load logs: (1054, "Unknown column 'endpoint' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
2025-07-28 10:18:25,844 ERROR: Failed to load logs: (1054, "Unknown column 'email' in 'field list'") [in c:\Users\PC\Desktop\CyberSystem\SchoolApplication\app.py:299]
